config/routes.rb

```diff
@@ -8,6 +8,6 @@
 
     get "translate", to: "translations#index"
     get "about", to: "static#about"
-    root to: "doctors#search"
+    root to: "doctors#search_root"
```

Comment:
Do you think it's acceptable to keep the ```search_root``` view with the button and get JS to reveal the button on load? 
Page-specific JS for this page confuses me, since there's nothing specific on the root page is there?